### ISSUE TITLE
Reading downloaded book makes crashing

### ISSUE THREAD

**Issue by Eveheeero** [Librum-Reader/Librum#124]
Save downloaded book to file works well, but reading downloaded book crashes (in both windows, linux)

**Comment by DavidLazarescu** [comment]
Hello, i suppose there was an issue with uploading the book to the cloud, so downloading gets an corrupted book. I would be able to check the book if you'd provide me the email you used to create your librum account.

**Comment by Eveheeero** [comment]
I created another Librum account for testing purposes (xhve00000@gmail.com) I always read docs in txt files, and if I remove them from my device and download and read them, it always crashes (different account, different OS, don't know about mac)

If I save the file to a file without reading it, it creates a Tex file, which has exactly the same content as the original txt file (not the format of the Tex file)

**Comment by Eveheeero** [comment]
I updated to the new version and tried logging in and out several times on multiple devices.

The epub file can be downloaded and viewed normally, but the txt file cannot be viewed after downloading.

Here's log in linux
```
qt.qpa.qgnomeplatform: Could not find color scheme  ""
terminate called after throwing an instance of 'mupdf::FzErrorUnsupported'
  what():  code=6: cannot find document handler for file: /home/eveheeero/.var/app/com.librumreader.librum/data/Librum-Reader/Librum/local_libraries/17326128032025319802/dd7847bd-c4d5-45bf-bb5c-48624c786667.Tex
```

**Comment by DavidLazarescu** [comment]
Thanks, the log you provided is very helpful. I'll be looking to fix it and include the fix in the next version.

**Comment by DavidLazarescu** [comment]
Could you provide a file that causes the issue you reported, so that I can replicate the error?

**Comment by Eveheeero** [comment]
The error occurs even in the simplest test.txt. This video demonstrates the error.
It's obviously reading the document from librum after downloading the file locally.
When saved as a file, the original file comes out fine.
(epub files don't seem to give this error).

https://youtu.be/-fN2wC3igkg

**Comment by DavidLazarescu** [comment]
Thanks for providing the example. I am able to replicate it and a fix for it will be in the next release.

**Comment by DavidLazarescu** [comment]
Just as a follow up. I have found and fixed the error in the following commit 6474703d9c7b923020ca22f5759bc23a04d99050
